.travis.yml

```diff
@@ -10,27 +10,44 @@ cache:
     - $HOME/.composer/cache
 
 php:
+  - 7.3
   - 7.2
   - 7.1
   - 7.0
 
 env:
   - WP_VERSION=latest WC_VERSION=latest
-  - WP_VERSION=latest WC_VERSION=3.4.5
-  - WP_VERSION=latest WC_VERSION=3.3.5
-  - WP_VERSION=latest WC_VERSION=3.2.6
+  - WP_VERSION=4.9.9 WC_VERSION=latest
+  - WP_VERSION=4.9.9 WC_VERSION=3.4.7
+  - WP_VERSION=4.9.9 WC_VERSION=3.3.5
+  - WP_VERSION=4.9.9 WC_VERSION=3.2.6
 
+# Considerations for the test matrix:
+#
+# - The "WC Requires At Least" is 3.2.6, so that's the lowest we need to go
+# - WordPress < 5.0 doesn't officially support PHP 7.3.
+# - WordPress 5.0 requires at least WooCommerce 3.5.1
 matrix:
   fast_finish: true
   include:
     - name: Coding Standards
       php: 7.2
-      env: WP_VERSION=trunk WC_VERSION=latest RUN_PHPCS=1
-    # Generate code coverage from PHP 7.1.
+      env: WP_VERSION=latest WC_VERSION=latest RUN_PHPCS=1
     - name: Code Coverage
       php: 7.2
-      env: WP_VERSION=latest WP_MULTISITE=0 WC_VERSION=latest RUN_CODE_COVERAGE=1
-
+      env: WP_VERSION=latest WC_VERSION=latest RUN_CODE_COVERAGE=1
+    - name: Bleeding Edge
+      php: 7.3
+      env: WP_VERSION=trunk WC_VERSION=latest
+  exclude:
+    - php: 7.3
```

Comment:
4.9.9 does't support 7.3.

Reply:
That's [the `exclude` node](https://docs.travis-ci.com/user/build-matrix/#excluding-jobs), where we can say "don't bother with this matrix combination"

Reply:
Missed that part - so, carry on :)